setup.py

```diff
@@ -4,7 +4,7 @@
 
 project_url = 'https://github.com/melexis/robot2rst'
 
-requires = ['robotframework', 'sphinxcontrib-robotdoc', 'mlx.traceability', 'mako']
+requires = ['robotframework', 'sphinxcontrib-robotdoc>=0.11.0,<0.12.0', 'mlx.traceability', 'mako']
```

Comment:
Dont we have in sphinx something like robot<1.11.99

Reply:
>=0.11.0,<=0.11.99 would do the same as my solution